Source/CarthageKit/Xcode.swift

```diff
@@ -1174,16 +1176,13 @@ public func nonDestructivelyStripArchitectures(_ frameworkURL: URL, _ architectu
 				$0.next() ?? ""
 			})
 
-			let suffix = zip(frameworkPathComponents, $0.pathComponents).drop(while: { $0 == $1 })
```

Comment:
This got fixed at https://github.com/Carthage/Carthage/commit/545be4dfee84b34c7d8d4064c5cd982f23a550cc#diff-f66644e8cfbcad3c3826395a690ddfbc.